recipes/llvmlite/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - python
     - setuptools
     - enum34  # [py2k]
-    - llvmdev ==5.0.0 
+    - llvmdev 
```

Comment:
I doubt the pin broke it as this is the latest version. So presumably it already passed with this version.